src/javascript/app_2/App/Components/Elements/PositionsDrawer/helpers/positions-helper.js

```diff
@@ -0,0 +1,43 @@
+import moment       from 'moment';
+import { localize } from '_common/localize';
+
+export const addCommaToNumber = (number) => number.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+
+export const getTimePercentage = (start_time, purchase_time, expiry_time) => {
```

Comment:
Would it be worth adding a check for the parameters + maybe we could add some unit tests for this function? `if (!start_time || ...`